CHANGELOG.md

```diff
@@ -51,6 +51,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Improve summary error message for invalid setting updates ([#4792](https://github.com/opensearch-project/OpenSearch/pull/4792))
 - Pass localNode info to all plugins on node start ([#7919](https://github.com/opensearch-project/OpenSearch/pull/7919))
 - Improved performance of parsing floating point numbers ([#7909](https://github.com/opensearch-project/OpenSearch/pull/7909))
+- Improved plugin support for repository-gcs by adding Google Application Default Credentials ([#8394](https://github.com/opensearch-project/OpenSearch/pull/8394))
```

Comment:
"Added support for Google Application Default Credentials in repository-gcs"

Reply:
Thanks, will change that as well.